packages/frontend/src/config/whitelabel.js

```diff
@@ -1,15 +1,11 @@
 import Environments from '../../../../features/environments.json';
-import environmentConfig from './configFromEnvironment';
+import { NEAR_WALLET_ENV } from './configFromEnvironment';
 
-const isWhitelabel = () => {
-    const { NEAR_WALLET_ENV } = environmentConfig;
-
-    return NEAR_WALLET_ENV === Environments.DEVELOPMENT ||
-        NEAR_WALLET_ENV === Environments.TESTNET ||
-        NEAR_WALLET_ENV === Environments.TESTNET_STAGING ||
-        NEAR_WALLET_ENV === Environments.MAINNET ||
-        NEAR_WALLET_ENV === Environments.MAINNET_STAGING;
-};
+const isWhitelabel = NEAR_WALLET_ENV === Environments.DEVELOPMENT
```

Comment:
Quite destructive change - if `isWhiteLabel` is used anywhere with old API - it will break the page.
For example [Transak](https://github.com/near/near-wallet/pull/2890) implementation uses old approach and will require update to support this.

It can be handled by pulling recent changes and updating Transak code btw.
But still need to make sure it is not broken anywhere else across the application

Reply:
As always with refactoring need to double check before merge.

Reply:
Fixed for Transak